02-FirstApplicationEnvironment/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ubuntu:16.04
+RUN apt-get update && apt-get install -y golang go git
```

Comment:
When i run `docker build .` I get the following error:

Reading state information...
E: Unable to locate package go
The command '/bin/sh -c apt-get update && apt-get install -y golang go git' returned a non-zero code: 100

I do not think these are the right package names.